rskj-core/build.gradle

```diff
@@ -62,6 +62,7 @@ ext {
             spongyCastleVer        : '1.52.0.0',
             bouncyCastleVer        : '1.59',
             ethereumLeveldbJniVer  : '1.18.3',
+            ethereumRocksdbJniVer  : '6.27.3',
```

Comment:
minor thing.. I thing we could call it simply `rocksdbJniVer` ? this one wasn't built by the ethereum team

Reply:
Done